w5e4.py

```diff
@@ -0,0 +1,102 @@
+# "w5e4"
+
+# connected vessels
+'''
+ Write a function called interleave that gets one or more iterable parameters, and returns a list of intertwined members.
+'''
+
+# function to interleave lists
+def interleave(*iterable):
+    longest = max(len(i) for i in iterable)
+    result = []
```

Comment:
result? pick a meaningful name

Reply:
I want to enjoy reading the code, its supposed to be as close to English a possible